examples/googmodule/tsconfig.json

```diff
@@ -1,4 +1,10 @@
 {
+    "compilerOptions": {
+        // Don't auto-discover @types since we are on typescript 2.7 in and
```

Comment:
need to test if this is actually the case with ts 2.8, 2.9 and 3.0

Reply:
added tests for different typescript versions but still not sure under what conditions typescript will auto-discover types